pkg/cmd/server/apis/config/types.go

```diff
@@ -1574,6 +1575,11 @@ type ServiceAccountControllerConfig struct {
 	ManagedNames []string
 }
 
+type DockerPullSecretControllerConfig struct {
+	// RegistryURLs is a list of urls that the docker pull secrets should be valid for.
+	RegistryURLs []string
```

Comment:
can we call this AdditionalRegistryURLs to match the controller field?

Reply:
> nit: optional: can we call this AdditionalRegistryURLs to match the controller field?

I think from the config side, the user see this as his spot to specify the registry urls, not to set extra ones.  From his point of view, we're the ones adding extras.  The API is for them, not us.